srcjs/init.js

```diff
@@ -12,8 +12,15 @@ $( document ).ready(function() {
   let config = setConfig();
   // create app instance
   app = new Framework7(config);
-  // init main view
-  mainView = app.views.create('.view-main');
+
+  var mainView = app.views.get(".view-main");
```

Comment:
specific to the router element. This allows inputs to correctly be bound whenever a page is mounted. Note that `keepalive` must be TRUE and `allowBrowserHistory` as well.